### DISCUSSION TITLE
How make plugins active

### DISCUSSION THREAD

**Discussion by SLV54** [n3d1117/chatgpt-telegram-bot#381] [Q&A]
Hi all

After downloading the archive with the bot, there is no plugins folder in the bot folder.
How to download the plugins folder?
Where I need to insert the line PLUGINS=weather,crypto,wolfram,spotify,ddg_web_search,ddg_image_search,ddg_translate)?
What else do i need to use plugins?

**Comment by n3d1117** [comment]
Hey @SLV54, first off make sure you clone the `feature/support-functions` branch, as the functionality is not yet available in `main` branch.
Then you can just add `PLUGINS=weather` to your `.env` file and it should work. Note that some plugins require additional configuration, check out the README at https://github.com/n3d1117/chatgpt-telegram-bot/tree/feature/support-functions#functions

**Reply by SLV54** [reply]
@n3d1117 Thanks for the quick response. Please explain how to clone the `feature/support-functions` branch

**Reply by n3d1117** [reply]
@SLV54 `git clone -b feature/support-functions https://github.com/n3d1117/chatgpt-telegram-bot.git`
